rpc_jobs/rpc_openstack.yml

```diff
@@ -49,6 +49,53 @@
     jobs:
       - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
 
+- project:
+     name: "rpc-openstack-queens-premerge"
```

Comment:
The indentation of all these projects appears to be off. I think this is the reason for the failure.